README.md

```diff
@@ -6,20 +6,21 @@ The Ziggurat implementation for Algorand's `algod` nodes.
 
 ## Getting started
 
-1. Clone this repository.
-2. Build [go-algorand](https://github.com/algorand/go-algorand) from source.
-3. Run the setup script:
+1. Ensure you have `rust` including its package manager `cargo` installed. You can verify that `cargo` is on your path by running `cargo version`.
+2. Clone this repository.
+3. Build [go-algorand](https://github.com/algorand/go-algorand) from source.
+4. Run the setup script:
 ```zsh
     tools/setup_env.sh
 ```
 
-In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH`
+In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH
```

Comment:
We still need this backtick, I believe: 
```suggestion
In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH`
```